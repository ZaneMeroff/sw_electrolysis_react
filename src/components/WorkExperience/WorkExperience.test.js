import React from 'react';
import { WorkExperience } from './WorkExperience';
import { shallow } from 'enzyme';

describe('WorkExperience', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<WorkExperience />);
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});
