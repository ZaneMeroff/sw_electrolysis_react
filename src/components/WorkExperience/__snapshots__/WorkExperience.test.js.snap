// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorkExperience should match the snapshot 1`] = `
<section
  className="work-experience-outer-container"
  id="experience-section"
>
  <div
    className="experience-container"
  >
    <div
      className="experience-heading-box"
    >
      <div
        className="experience-heading-border"
      >
        <h3
          className="experience-heading"
        >
          Experience
        </h3>
      </div>
    </div>
    <div
      className="experience-right-container"
    >
      <div
        className="job-container"
        key="Hylio"
      >
        <p
          className="experience-title"
        >
          Hylio
        </p>
        <i
          className="experience-position"
        >
          Software Developer (full-time, remote)
        </i>
        <i
          className="experience-time"
        >
          January 2021 - present
        </i>
        <ul
          className="description-list"
        >
          <li
            className="experience-item"
            key="Build eye catching, accessible, and responsive front end software using Vue.js with Quasar and Electron"
          >
            Build eye catching, accessible, and responsive front end software using Vue.js with Quasar and Electron
          </li>
          <li
            className="experience-item"
            key="Work closely with marketing team to develop new design concepts and bring Figma wire frame layouts to life for web and native platforms"
          >
            Work closely with marketing team to develop new design concepts and bring Figma wire frame layouts to life for web and native platforms
          </li>
          <li
            className="experience-item"
            key="Employ agile methodologies throughout the development process, including stand-ups, sprint planning, backlog grooming, scrum framework, and retrospectives"
          >
            Employ agile methodologies throughout the development process, including stand-ups, sprint planning, backlog grooming, scrum framework, and retrospectives
          </li>
          <li
            className="experience-item"
            key="Accomplishments include successfully completing the Hylio Drone Configurator app (sole programmer) from conception to deployment on AWS using Elastic Beanstalk"
          >
            Accomplishments include successfully completing the Hylio Drone Configurator app (sole programmer) from conception to deployment on AWS using Elastic Beanstalk
          </li>
        </ul>
      </div>
      <div
        className="job-container"
        key="B-Line Express"
      >
        <p
          className="experience-title"
        >
          B-Line Express
        </p>
        <i
          className="experience-position"
        >
          Software Developer (part-time, remote)
        </i>
        <i
          className="experience-time"
        >
          June 2020 - present
        </i>
        <ul
          className="description-list"
        >
          <li
            className="experience-item"
            key="Create modern apps using Vue.js and develop solutions to enhance UI/UX, improve processes, and push the boundaries of what is possible from front end technologies"
          >
            Create modern apps using Vue.js and develop solutions to enhance UI/UX, improve processes, and push the boundaries of what is possible from front end technologies
          </li>
          <li
            className="experience-item"
            key="Involved in all steps of app development, including conception, design, development, release, and iterative improvement"
          >
            Involved in all steps of app development, including conception, design, development, release, and iterative improvement
          </li>
          <li
            className="experience-item"
            key="Accomplishments include successfully leading development team to meet deadline for creating the membership management module for Porsche Club of America’s GloveBox application"
          >
            Accomplishments include successfully leading development team to meet deadline for creating the membership management module for Porsche Club of America’s GloveBox application
          </li>
        </ul>
      </div>
      <div
        className="job-container"
        key="Boom Supersonic"
      >
        <p
          className="experience-title"
        >
          Boom Supersonic
        </p>
        <i
          className="experience-position"
        >
          Software Developer Internship
        </i>
        <i
          className="experience-time"
        >
          August 2020 - November 2020
        </i>
        <ul
          className="description-list"
        >
          <li
            className="experience-item"
            key="Implemented test driven development for flight software in a legacy code base with Vue.js"
          >
            Implemented test driven development for flight software in a legacy code base with Vue.js
          </li>
          <li
            className="experience-item"
            key="Built visual representations of dynamic data for TelViz flight management tool using D3 JavaScript library with SVG"
          >
            Built visual representations of dynamic data for TelViz flight management tool using D3 JavaScript library with SVG
          </li>
          <li
            className="experience-item"
            key="Worked to identify and fix bugs affecting UI/UX, functionality, and performance"
          >
            Worked to identify and fix bugs affecting UI/UX, functionality, and performance
          </li>
          <li
            className="experience-item"
            key="Accomplishments include creating unit regression and end to end testing coverage across entire TeCSys single page web application using Jest and Cypress with CircleCI for continuous integration"
          >
            Accomplishments include creating unit regression and end to end testing coverage across entire TeCSys single page web application using Jest and Cypress with CircleCI for continuous integration
          </li>
        </ul>
      </div>
      <div
        className="job-container"
        key="Adrift Tiki Bar & Restaurant"
      >
        <p
          className="experience-title"
        >
          Adrift Tiki Bar & Restaurant
        </p>
        <i
          className="experience-position"
        >
          General Manager
        </i>
        <i
          className="experience-time"
        >
          March 2016 - March 2019
        </i>
        <ul
          className="description-list"
        >
          <li
            className="experience-item"
            key="Sole general manager responsible for overseeing two million dollar gross annual operation"
          >
            Sole general manager responsible for overseeing two million dollar gross annual operation
          </li>
          <li
            className="experience-item"
            key="Reduced annual labor costs by 8% within first year"
          >
            Reduced annual labor costs by 8% within first year
          </li>
          <li
            className="experience-item"
            key="Increased annual gross sales by 15% within first year"
          >
            Increased annual gross sales by 15% within first year
          </li>
          <li
            className="experience-item"
            key="Created multiple successful marketing campaigns and special events"
          >
            Created multiple successful marketing campaigns and special events
          </li>
          <li
            className="experience-item"
            key="Created SOPs for all positions and comprehensive onboard training programs"
          >
            Created SOPs for all positions and comprehensive onboard training programs
          </li>
        </ul>
      </div>
      <div
        className="job-container"
        key="Camp Watcha Wanna Do"
      >
        <p
          className="experience-title"
        >
          Camp Watcha Wanna Do
        </p>
        <i
          className="experience-position"
        >
          Board of Directors / Event Planner
        </i>
        <i
          className="experience-time"
        >
          June 2010 - March 2016
        </i>
        <ul
          className="description-list"
        >
          <li
            className="experience-item"
            key="Non-profit organization bettering the quality of life for adolescent cancer survivors"
          >
            Non-profit organization bettering the quality of life for adolescent cancer survivors
          </li>
          <li
            className="experience-item"
            key="Organized and ran fundraising events generating over one million dollars"
          >
            Organized and ran fundraising events generating over one million dollars
          </li>
          <li
            className="experience-item"
            key="Led Board of Directors and served as Camp Counselor"
          >
            Led Board of Directors and served as Camp Counselor
          </li>
          <li
            className="experience-item"
            key="Created detailed outlines for camp procedures/activities and board agendas"
          >
            Created detailed outlines for camp procedures/activities and board agendas
          </li>
        </ul>
      </div>
    </div>
  </div>
  <div
    className="tech-experience-container"
    id="skills-section"
  >
    <div
      className="experience-heading-box"
    >
      <div
        className="experience-heading-border"
      >
        <h3
          className="experience-heading"
        >
          Tech Skills
        </h3>
      </div>
    </div>
    <div
      className="tech-skills-container"
    >
      <ul
        className="tech-skills-bullets"
      >
        <li
          className="experience-item"
          key="JavaScript"
        >
          JavaScript
        </li>
        <li
          className="experience-item"
          key="TypeScript"
        >
          TypeScript
        </li>
        <li
          className="experience-item"
          key="HTML5"
        >
          HTML5
        </li>
        <li
          className="experience-item"
          key="CSS3"
        >
          CSS3
        </li>
        <li
          className="experience-item"
          key="Vue.js"
        >
          Vue.js
        </li>
        <li
          className="experience-item"
          key="Vuex"
        >
          Vuex
        </li>
        <li
          className="experience-item"
          key="Vue Router"
        >
          Vue Router
        </li>
        <li
          className="experience-item"
          key="React"
        >
          React
        </li>
        <li
          className="experience-item"
          key="React Native"
        >
          React Native
        </li>
        <li
          className="experience-item"
          key="React Router"
        >
          React Router
        </li>
        <li
          className="experience-item"
          key="Redux"
        >
          Redux
        </li>
        <li
          className="experience-item"
          key="Node.js"
        >
          Node.js
        </li>
        <li
          className="experience-item"
          key="Electron"
        >
          Electron
        </li>
        <li
          className="experience-item"
          key="Elastic Beanstalk"
        >
          Elastic Beanstalk
        </li>
        <li
          className="experience-item"
          key="Docker"
        >
          Docker
        </li>
        <li
          className="experience-item"
          key="D3.js"
        >
          D3.js
        </li>
        <li
          className="experience-item"
          key="Photoshop"
        >
          Photoshop
        </li>
        <li
          className="experience-item"
          key="jQuery"
        >
          jQuery
        </li>
        <li
          className="experience-item"
          key="TravisCI"
        >
          TravisCI
        </li>
        <li
          className="experience-item"
          key="CircleCI"
        >
          CircleCI
        </li>
        <li
          className="experience-item"
          key="Puppeteer"
        >
          Puppeteer
        </li>
        <li
          className="experience-item"
          key="Cypress"
        >
          Cypress
        </li>
        <li
          className="experience-item"
          key="Quasar"
        >
          Quasar
        </li>
        <li
          className="experience-item"
          key="Stylus"
        >
          Stylus
        </li>
        <li
          className="experience-item"
          key="MD Bootstrap"
        >
          MD Bootstrap
        </li>
        <li
          className="experience-item"
          key="SCSS / SASS"
        >
          SCSS / SASS
        </li>
        <li
          className="experience-item"
          key="Jest / Enzyme"
        >
          Jest / Enzyme
        </li>
        <li
          className="experience-item"
          key="Mocha / Chai"
        >
          Mocha / Chai
        </li>
        <li
          className="experience-item"
          key="NPM / Yarn"
        >
          NPM / Yarn
        </li>
        <li
          className="experience-item"
          key="Git / GitHub"
        >
          Git / GitHub
        </li>
        <li
          className="experience-item"
          key="Bitbucket"
        >
          Bitbucket
        </li>
      </ul>
    </div>
  </div>
  <div
    className="experience-container"
  >
    <div
      className="experience-heading-box"
    >
      <div
        className="experience-heading-border"
      >
        <h3
          className="experience-heading"
        >
          Education
        </h3>
      </div>
    </div>
    <div
      className="experience-right-container"
    >
      <h3
        className="experience-title"
      >
        Turing School of Software & Design
      </h3>
      <i
        className="experience-position"
      >
        Graduate Front End Web Development Program
      </i>
      <i
        className="experience-time"
      >
        September 2019 - April 2020
      </i>
      <ul
        className="description-list"
      >
        <li
          className="experience-item"
          key="Built 14 unique web applications from scratch with vanilla JS, React, & React Native"
        >
          Built 14 unique web applications from scratch with vanilla JS, React, & React Native
        </li>
        <li
          className="experience-item"
          key="Created thorough unit testing suites for every application"
        >
          Created thorough unit testing suites for every application
        </li>
        <li
          className="experience-item"
          key="Programmed for team projects of four people, paired projects, and solo projects"
        >
          Programmed for team projects of four people, paired projects, and solo projects
        </li>
        <li
          className="experience-item"
          key="Focused on JavaScript fundamentals for 12 weeks"
        >
          Focused on JavaScript fundamentals for 12 weeks
        </li>
        <li
          className="experience-item"
          key="Completed 1,200 hours building applications with front end and back end technologies"
        >
          Completed 1,200 hours building applications with front end and back end technologies
        </li>
        <li
          className="experience-item"
          key="Dedicated 600 hours working strictly with React, React Native, Router, and Redux"
        >
          Dedicated 600 hours working strictly with React, React Native, Router, and Redux
        </li>
      </ul>
    </div>
  </div>
</section>
`;
